test/e2e/handler/webhook_test.go

```diff
@@ -86,4 +86,20 @@ var _ = Describe("Validation Admission Webhook", func() {
 			Expect(err).To(MatchError("admission webhook \"nodenetworkconfigurationpolicies-create-validate.nmstate.io\" denied the request: failed to admit NodeNetworkConfigurationPolicy this-is-longer-than-sixty-three-characters-hostnames-bar-bar.com: message: invalid policy name: \"this-is-longer-than-sixty-three-characters-hostnames-bar-bar.com\": must be no more than 63 characters. "))
 		})
 	})
+	Context("When a policy is created with capture field", func() {
+		BeforeEach(func() {
+			By("Create a policy")
```

Comment:
Should we add an expect that would check the capture field is indeed set?

Reply:
It doe not create the capture field, we try to create it later on doing a NNCP modification, that's how I check that the webhook is working without adding capture field to TestPolicy, so it play nice with the rest of the tests.

Reply:
Ahh, I see. I find the Context a bit misleading then.
Could it be `When a policy capture field is updated`.